src/script.js

```diff
@@ -0,0 +1,45 @@
+// Creates the menu view with buttons for adding items to the order
+$(document).ready(function(){
+  $('#add-payment-form').hide();
+  $.getJSON( "menu.json", function( data ) {
+    var myShop = new CoffeeShop(data[0]);
+    $( "#resto-name" ).text(myShop.shopName);
+    $( "#resto-address" ).text(myShop.address);
+    
+    var items = [];
+
+    $.each( myShop.prices, function( key, val ) {
+      items.push( "<li><button type='button' value='"+key+"'>" + key + ": " + val + "</button></li>" );
+    });
+   
+    $( "<ul/>", {
+      "class": "product-buttons",
+      html: items.join( "" )
+    }).appendTo( ".left" );
+
+    $(".product-buttons").find('button').click( function () {
+      myShop.addItem($(this).val());
+      myShop.showOrder();
+
+      var list = "<ul class='receipt-items'>";
```

Comment:
Generally speaking it's not a great idea to be mixing HTML generation inside of your JS (although I'm aware this is often done in the real world) - instead using something like mustache would allow you to keep these two things seperate.